src/root.jsx

```diff
@@ -26,6 +26,8 @@ class RootRecord extends quip.apps.RootRecord {
         speechTitle1: quip.apps.RichTextRecord,
         speaker2: quip.apps.RichTextRecord,
         speechTitle2: quip.apps.RichTextRecord,
+        speaker3: quip.apps.RichTextRecord,
```

Comment:
Gee, I don't know about naming it as `speaker3` internally, if we're exposing it as "Backup speaker" externally. In the long run, it would be great if we support customizable agendas, and some clubs can configure 3 speakers as necessary. That requires re-architecting the data record layout, though, and is something I've been thinking of for months now 😄 

Is there anything morally wrong with being consistent and calling it `backupSpeaker` everywhere?